src/network/RTypeClient.cpp

```diff
@@ -0,0 +1,166 @@
+/*
+** EPITECH PROJECT, 2024
+** RType
+** File description:
+** RTypeClient
```

Comment:
Could you add the suffix like for other files ? RTypeClient.cpp in this case

Reply:
Done